rpc/SynchronizedStreamObserver.java

```diff
@@ -0,0 +1,19 @@
+package grakn.client.rpc;
```

Comment:
This will fail `checkstyle_test` unless you add a license header at the top of the file.